tensorboard/components/vz_line_chart/vz-line-chart.ts

```diff
@@ -590,18 +628,28 @@ class LineChart {
       yDomain = this._defaultYRange;
     } else {
       // Generate a reasonable range.
-      const accessor = this.getAccessor();
-      let datasetToValues: (d: Plottable.Dataset) => number[] = (d) => {
-        return d.data().map((x) => accessor(x, -1, d));
+      const accessors = this.getAccessorsForComputingYRange();
+      let datasetToValues: (d: Plottable.Dataset) => number[][] = (d) => {
+        return d.data().map(x => accessors.map(accessor => accessor(x, -1, d)));
       };
-      let vals = _.flatten(this.datasets.map(datasetToValues));
+      let vals = _.flattenDeep<number>(this.datasets.map(datasetToValues));
       vals = vals.filter((x) => x === x && x !== Infinity && x !== -Infinity);
```

Comment:
tangent: Doesn't have to happen in this PR, but it bugs me that this isn't just `vals.filter(x => isFinite(x))`…